test/system_SUITE.erl

```diff
@@ -355,6 +361,59 @@ roundtrip(OpenConf, Body) ->
     ?assertEqual([Body], amqp10_msg:body(OutMsg)),
     ok.
 
+filtered_roundtrip(OpenConf) ->
+    filtered_roundtrip(OpenConf, <<"banana">>).
+
+filtered_roundtrip(OpenConf, Body) ->
```

Comment:
There are dialyzer errors in this function

```  Checking whether the PLT /Users/knilsson/code/rabbitmq/rabbitmq-public-umbrella-master/deps/amqp10_client/.amqp10_client.plt is up-to-date... yes
  Proceeding with analysis...
system_SUITE.erl:306: Function filtered_roundtrip_service_bus/1 has no local return
system_SUITE.erl:364: Function filtered_roundtrip/1 has no local return
system_SUITE.erl:367: Function filtered_roundtrip/2 has no local return
system_SUITE.erl:400: Call to missing or unexported function amqp10_client:attach_receiver_link/6
system_SUITE.erl:405: Binary construction will fail since the value field Now2Binary in segment Now2Binary:8/integer-unit:1 has type binary()```

Reply:
Looks like you haven't exported `amqp10_client:attach_receiver_link/6`